message/tests.py

```diff
@@ -0,0 +1,83 @@
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='Hello this is the test message1 from owner to day care')
+
+
+@pytest.fixture
+def daycare_reply_to_dogonwer_message1():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message from daycare to owner')
+
+
+@pytest.fixture
+def daycare_message2():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=2),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='Hello this is the test message2 from daycare to owner')
+
+
+@pytest.fixture
+def dogowner_reply_to_daycare_message2():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=2),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='This is reply to second message from dogowner to daycare')
+
+
+@pytest.mark.django_db()
+class TestMessageModel:
+    def test_create_message(self, dogowner_message1):
+        assert dogowner_message1 in Message.objects.all()
+
+    def test_delete_message(self, dogowner_message1):
+        assert dogowner_message1 in Message.objects.all()
+
+        dogowner_message1.delete()
+        assert dogowner_message1 not in Message.objects.all()
+
+    def test_chat_between_dogowner_daycare(self, dogowner_message1, daycare_reply_to_dogonwer_message1,
+                                           daycare_message2, dogowner_reply_to_daycare_message2):
+        firstChat = Message.get_chat_between_dogowner_daycare(dogowner_message1.dogowner_id,
+                                                              dogowner_message1.daycare_id)
+        secondChat = Message.get_chat_between_dogowner_daycare(daycare_message2.dogowner_id,
+                                                               daycare_message2.daycare_id)
+
+        assert dogowner_message1 in firstChat
+        assert daycare_reply_to_dogonwer_message1 in firstChat
+        assert daycare_message2 in secondChat
+        assert dogowner_reply_to_daycare_message2 in secondChat
+
+    def test_dog_owner_contacts(self, dogowner_message1, daycare_reply_to_dogonwer_message1,
+                                daycare_message2, dogowner_reply_to_daycare_message2):
+        dogowner1contacts = Message.get_all_dogowner_contacts(dogowner_message1.dogowner_id)
+        dogowner2contacts = Message.get_all_dogowner_contacts(daycare_message2.dogowner_id)
+
+        assert dogowner_message1.daycare_id in dogowner1contacts
+        assert daycare_message2.daycare_id in dogowner2contacts
```

Comment:
I don't really get what you're testing here, 
but either way, it seems like you test two unrelated objects on the same test

Reply:
added more daycare contacts for dogowner#1